prov/efa/src/rxr/rxr_pkt_type_data.c

```diff
@@ -62,7 +64,12 @@ ssize_t rxr_pkt_send_data(struct rxr_ep *ep,
 	data_pkt = (struct rxr_data_pkt *)pkt_entry->pkt;
 	data_pkt->hdr.seg_size = payload_size;
 
-	copied_size = rxr_copy_from_tx(data_pkt->data, payload_size, tx_entry, tx_entry->bytes_sent);
+	copied_size = ofi_copy_from_hmem_iov(data_pkt->data,
```

Comment:
can we keep the function ```rxr_copy_from_tx``` and call ```ofi_copy_from_hmem_iov``` from it? Also ```ofi_copy_from_hmem_iov``` use function pointer ever for system memory. Would that have some performance impact? cache miss, etc?

Reply:
`rxr_copy_from_tx()` would do nothing more than calling the hmem copy functions, so I did not see a point in keeping it, particularly given the the hmem methods can handle system memory too. Is there a reason why you think we would need the wrapper?

For the performance discussion, check out https://github.com/ofiwg/libfabric/issues/5928. We can chat about this offline too some more, but in this particular case, the use of the dispatch tables give us the benefit of having a single build to serve multiple environments.

Reply:
I just approved it. 

I am  still concerned about using function pointer even for host memory, but we will find out the performance impact of it once it is merged and tested.

I also think the function name ```ofi_copy_from_hmem_iov``` is misleading, because it works not only on hmem but also system memory, but it is beyond the scope of this commit.